src/plugins/ExecuteJob/templates/deepforge.ejs

```diff
@@ -37,6 +37,8 @@ function _Line:__init(graphId, name, opts)
 end
 
 function _Line:add(x, y)
+    assert(type(x) == "number", "add expects 'x' to be a number")
```

Comment:
Yep, I like that. I was going off of the lua documentation, but their errors are inherently bad.